modules/ROOT/pages/security/ssl-framework.adoc

```diff
@@ -13,138 +13,76 @@ The SSL framework provides support for securing the following Neo4j communicatio
 This page describes how to set up SSL within your environment, how to view, validate, and test the certificates.
 
 [[ssl-providers]]
-== SSL providers
+== SSL Providers
 
 The secure networking in Neo4j is provided through the Netty library, which supports both the native JDK SSL provider as well as Netty-supported OpenSSL derivatives.
+Each version of Neo4j ships with a version of Netty, and Netty requires a specific version of the `netty-tcnative` library for compatibility.
 
 Follow these steps to use OpenSSL:
 
-- Install a suitable dependency into the `plugins/` folder of Neo4j.
+* Install a suitable `netty-tcnative` dependency into the `plugins/` folder of Neo4j.
+** Dependencies can be downloaded from https://netty.io/wiki/forked-tomcat-native.html.
+** Which `netty-tcnative` version you need depends upon the Neo4j version.
+For versioning details, see the <<table, Netty support per Neo4j version>> table.
+Make sure to install a build variant that matches your OS and architecture.
+* Set `xref:configuration/configuration-settings.adoc#config_dbms.netty.ssl.provider[dbms.netty.ssl.provider]=OPENSSL`.
+* Restart Neo4j.
 
 [NOTE]
 ====
-Dependencies can be downloaded from https://netty.io/wiki/forked-tomcat-native.html.
-Which dependencies you need depends upon the Neo4j version.
-Each version of Neo4j ships with a version of Netty and Netty requires specific tcnative versions.
-Make sure to install the version that matches your OS processor.
-For more details, see the <<table, Netty support per Neo4j version>>.
+Using non-static versions of tcnative will require installation of platform-specific dependency libraries as described in https://netty.io/wiki/forked-tomcat-native.html.
```

Comment:
In the `ssl-fips-compatibility.adoc`, 'dynamic versions' are used. Should we change here 'non-static' to 'dynamic'?